use-cases/credit_risk/risk_bucketing.ipynb

```diff
@@ -0,0 +1,932 @@
+      "text/plain": [
+       "<Figure size 1000x800 with 4 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.figure(figsize=(10, 8))\n",
+    "k = 0\n",
+    "cols = numerical_credit.columns\n",
+    "for i, j in zip(range(len(cols)), cols):\n",
+    "    k +=1\n",
+    "    plt.subplot(2, 2, k)\n",
+    "    plt.hist(numerical_credit.iloc[:, i])\n",
+    "    plt.title(j)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The Elbow Method\n",
```

Comment:
Since you've added this notebook to the RST index file, in order to have this notebook rendered correctly on the website, use H2-level headings with 2 pound symbols instead of 1 for all sub-headings (`## The Elbow Method`, `## Silhouette Scores`, `## Gap Analysis`, `## K-Means Clustering`). A notebook needs a single H1-header which you will want to be `# Risk Bucketing`, the title of the notebook.

Reply:
Hi jkroll-aws,

As far as I'm aware I have done that (single pound for the single H1-header, double pound for all sub-headings).  Have you spotted somewhere I have missed this?

Reply:
Ah I see, that was my mistake in a manual search. Please disregard, the headings look correct.